src/graphics.c

```diff
@@ -616,10 +632,25 @@ void graphics_fill_screen( display_context_t disp, uint32_t c )
 }
 
 /**
- * @brief Draw a character to the screen using the built-in font
+ * @brief Set the current font. Should be set before using any of the draw function.
  *
- * @todo Need a set font mechanism.
- * @todo Need to look up the width and height from the font instead of assuming it is 8x8.
+ * @param[in] font
+ *        Sprite font to be used. Should be using a standard font sprite (containing every ASCII
+ * character) and with a transparent background, and be loaded on mksprite with width and height
+ * of 16.
+ * @param[in] tab_width
+ *        Width of each \\t used. Is multiplied by the width of each character.
+ */
+void graphics_set_font( sprite_t *font, int tab_width )
```

Comment:
Would you mind calling this `graphics_set_font_sprite`? I think eventually we want to a have a public `font_t` type (maybe even shared between graphics.c and rdp.c) that can be configured with a set of `font_*` functions, maybe has its own serialization functions and tools, and I'd like to keep the name `graphics_set_font` available for that scenario. This API would then remain as a quick fallback for backward compatibilty, so mentioning `sprite` in the name makes it clear that it's for the cases where you do have a `sprite_t` around.